src/node.cc

```diff
@@ -1174,7 +1175,7 @@ InitializeOncePerProcessInternal(const std::vector<std::string>& args,
 
     // Ensure CSPRNG is properly seeded.
     CHECK(crypto::CSPRNG(nullptr, 0).is_ok());
-
+#endif // defined(OPENSSL_IS_BORINGSSL)
     V8::SetEntropySource([](unsigned char* buffer, size_t length) {
```

Comment:
Is the V8 entropy source being changed intentionally here? If so, this `#endif` should likely be before the `CHECK(...)` above. If not, it should likely be after the call to `V8::SetEntropySource()`.